src/main/java/pl/jw/stronazformularzem/MailService.java

```diff
@@ -0,0 +1,40 @@
+package pl.jw.stronazformularzem;
+
+import jakarta.mail.MessagingException;
+import jakarta.mail.internet.MimeMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MailService {
+
+    private static final Logger logger = LoggerFactory.getLogger(MailService.class);
+
+    private JavaMailSender javaMailSender;
+
+    public MailService(JavaMailSender javaMailSender) {
+        this.javaMailSender = javaMailSender;
+    }
+
+    public void sendMail(String sender, String email, String content) {
+        logger.debug("Wysyłam maila od {}", sender);
+
+        MimeMessage mimeMessage = javaMailSender.createMimeMessage();
+        MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, "utf-8");
+        String htmlMsg = "<h3>Hello World!</h3>";
+        try {
+            helper.setSubject(sender);
+            helper.setFrom(email);
+            helper.setTo("jacek.wojtuszek@gmail.com");
+            helper.setText(content, true);
+            javaMailSender.send(mimeMessage);
```

Comment:
Tutaj powinno to wyglądać trochę inaczej.
Wiadomość wysyłamy z naszej skrzynki, de facto do samych siebie, a dodatkowo powinno być ustawione pole reply-to, które powinno wskazywać adres email podany w formularzu kontaktowym